reddit.js

```diff
@@ -2,4 +2,65 @@ var reddit = require("./data.json")
 
 // Write your solutions below
 
-console.log(reddit["data"]);
\ No newline at end of file
+// console.log(reddit["data"]);
+
+
+// Print all reddit titles using forEach: 
+var array = reddit.data.children;
+
+
+
+
+array.forEach(function (array) {
+	console.log(array.data.title);
+ });
+
+console.log("----------------------break----------------------")
+
+// Return permalinks using map
+// var links = array.forEach(function (array) {
+// 	console.log(array.data.permalink);	
+// });
+
+var permalinks = array.map(function (link) {
```

Comment:
Same case as with the above, ```child``` or ```post``` or ```object``` would be a more appropriate placeholder name here than ```link```.